style.css

```diff
@@ -253,12 +269,86 @@ p {
   margin-bottom: -3rem;
 }
 
+.contact {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  height: 100vh;
+}
+
 .contact p {
-  border: 2px solid red;
   font-family: 'Inter', sans-serif;
   font-weight: 700;
   font-size: 40px;
   text-align: center;
   color: #172b4d;
-  margin: 40px 25px;
+  margin: 120px 25px 50px 25px;
+  padding-top: 56px;
+}
+
+fieldset {
+  border: 0;
+  display: flex;
+  flex-direction: column;
+  gap: 3rem;
+  margin: 2rem auto;
+  width: 90%;
+}
+
+input {
+  border: 1px solid #d0d9d4;
+  border-radius: 4px;
+  padding: 21px;
+  height: 40px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  align-items: center;
+}
+
+textarea {
+  max-height: 140px;
+  padding: 13px;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 15px;
+  line-height: 24px;
+  color: #091e42;
+  background: #fff;
+  border: 1px solid #60c095;
+  border-radius: 4px;
+}
+
+.btn {
+  display: block;
+  padding: 12px;
+  width: 129px;
+  height: 48px;
+  background: #36b37f;
+  border-radius: 4px;
+  margin: 0 auto;
+  border: 0;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 17px;
+  line-height: 24px;
+  align-items: center;
+  text-align: center;
+  letter-spacing: 0.03em;
+  color: #fff;
+}
+
+.social-icons ul {
+  display: flex;
+  justify-content: center;
+  gap: 20px;
+  margin-top: 15rem;
+  list-style: none;
+}
+
+.email-paragraph {
```

Comment:
- [ ] Kindly add the margin-bottom below as per the Figma design :

your design:
![Screenshot from 2023-01-18 02-25-14](https://user-images.githubusercontent.com/97834160/213046688-15532171-344f-4630-99cd-ad688770495f.png)


Figma:

![img](https://user-images.githubusercontent.com/97834160/213047685-23499d0e-e1fc-4233-92fe-a674c281792f.png)